doc/getting-started/connections.rst

```diff
@@ -0,0 +1,76 @@
+Connections
+===========
+
+PyVista is a powerful tool that researchers can harness to create compelling,
+integrated visualizations of large datasets in an intuitive, Pythonic manner.
+
+PyVista is used across science and engineering disciplines by a diverse
+community to visualize data/models in 3D, generate publication-quality
+illustrations, automate analysis workflows, and build custom applications
+leveraging PyVista's 3D capabilities.
+
+Here are a few open-source projects that leverage PyVista:
+
+
+Engineering
+-----------
+
+* pyleecan_: Electrical engineering open-source software providing a user-friendly, unified, flexible simulation framework for the multiphysic design and optimization of electrical machines and drives.
+* pymapdl_: Pythonic interface to Ansys MAPDL.
+* pyprocar_: A Python library for electronic structure pre/post-processing.
+* AeroSandbox_: Plug-and-play analysis tools for aerodynamics, propulsion, structures, trajectory design, and much more.
+* Ptera_: Software for analyzing flapping-wing flight.
+
+
+Geosciences
+-----------
+
+* GemPy_: 3D structural geological modeling software using PyVista for 3D visualization and interaction.
+* gemgis_: Spatial data processing for geomodeling.
+* gimli_: Geophysical inversion and modeling library.
+* discretize_: Discretization tools for finite volume and inverse problems.
+* SEGY-SAK_: SEGY Swiss Army Knife for Seismic Data.
+* subsurface_: Core data exchange library for subsurface science and engineering
+* Flownet_: Data-Driven Reservoir Predictions.
+* PVGeo_: Python package of VTK-based algorithms to analyze geoscientific data and models. PyVista is used to make the inputs and outputs of PVGeo's algorithms more accessible.
+* omfvista_: 3D visualization for the Open Mining Format (omf). PyVista provides the foundation for this library's visualization.
+
+
+Many More
+---------
+
+* VesselVio_: Analysis and visualization of segmented vasculature datasets.
+* pymeshfix_: Python/Cython wrapper of Marco Attene's wonderful, award-winning MeshFix_ software.
+* tetgen_: Python Interface to Hang Si's C++ TetGen Library.
+* itkwidgets_: Interactive Jupyter widgets to visualize images, point sets, and meshes in 2D and 3D. Supports all PyVista mesh types.
+* mne-python_: MNE: Magnetoencephalography (MEG) and Electroencephalography (EEG) in Python.
+* pyntcloud_: Python library for working with 3D point clouds.
+* DiSECt_: Differentiable Cutting Simulator.
+* torch-points3d_: Pytorch framework for doing deep learning on point clouds.
+* pyprocar_: A Python library for electronic structure pre/post-processing
```

Comment:
`pyprocar` was already featured in the Engineering section, which is actually incorrect :P
```suggestion
* pyprocar_: A Python library for electronic structure pre/post-processing.
```